client/server/render/index.js

```diff
@@ -232,19 +234,16 @@ export function serverRender( req, res ) {
 	if ( context.store ) {
 		attachHead( context );
 
-		const cacheableReduxSubtrees = [ 'documentHead' ];
 		const isomorphicSubtrees = context.section?.isomorphic ? [ 'themes', 'ui' ] : [];
 
-		const reduxSubtrees = [ ...cacheableReduxSubtrees, ...isomorphicSubtrees ];
+		const reduxSubtrees = [ 'documentHead', ...isomorphicSubtrees ];
```

Comment:
What is the `documentHead` about, do you know? Is it somehow storing DOM elements in redux or am I completely misunderstanding this?

Reply:
I don't think it's the DOM elements, but some precomputed info about what goes into `<head>`